### ISSUE TITLE
`Nylas::CalendarCollection.availability` returns `Nylas::InvalidRequest` when `:round_robin` parameter is `nil`

### ISSUE THREAD

**Issue by kwayebopp** [nylas/nylas-ruby#373]
**Describe the bug**
Using `nil` as the value for the `:round_robin` parameter returns a `Nylas::InvalidRequest` error.

**To Reproduce**
Simply try calling the `availability` function on a `Nylas::CalendarCollection`. After looking through the codebase, [`:round_robin` appears to be `nil` by default](https://github.com/nylas/nylas-ruby/blob/107d35d2ef287056f7387a500143dce62f984581/lib/nylas/calendar_collection.rb#L20), so I think the bug should surface with whatever parameters you use.
```
nylas =  Nylas::API.new(...)
nylas.calendars.availability(
  #params go here
)
```

**Expected behavior**
[As per the developer docs](https://developer.nylas.com/docs/api/#post/calendars/availability), I expected that using `round_robin: nil` would return collective availability. 

**SDK Version:**
 5.11.0

**Additional context**

**Comment by mrashed-dev** [comment]
@kwayebopp thanks for opening this issue, we will take a look and provide an update as soon as we can!

**Comment by mrashed-dev** [comment]
Thanks for your patience @kwayebopp. After talking to the availability team, it seems the documentation is off. `round_robin` is optional, but `null` is not a valid value. Omitting the `round_robin` field will yield a successful result with collective availability. Sending `null` will return an error.

I opened a PR to address this, now we just omit the field if it's left to `nil`. We also opened an internal ticket to fix the docs to reflect this. Thanks again for spotting and reporting this issue!
